### PR TITLE
DEVDOCS-5131: [update] GQLSF Code Samples, add HTTP config

### PR DESCRIPTION
**Pull request by slsriehl**
# [DEVDOCS-5131]

## What changed?
Provide a bulleted list in the present tense
* add http config to code samples

## Anything else?
Add related PRs, salient notes, ticket numbers, etc.

ping @bc-andreadao

[DEVDOCS-5131]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5131?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/api-docs/storefront/graphql/graphql-api-examples.mdx [hunk 1]
```diff
@@ -4,12 +4,47 @@ keywords: storefront, graphql
...
+  .catch(error => console.error(error));
+```
+
+To use this API server-to-server, use the following HTTP configuration:
+
+```http filename="Server-to-server GraphQL Storefront HTTP configuration" showLineNumbers copy
+  POST https://{{storeDomain}}/graphql
+  Authorization: Bearer {{storefrontOrCustomerImpersonationToken}}
+  X-BC-Customer-ID: 123 # optional once a customer is signed in
+  Accept: application/json
```
Comment (bc-andreadao) [inline docs/api-docs/storefront/graphql/graphql-api-examples.mdx:45]: @slsriehl It looks like `Accept` is not needed, not sure why that is
no accept header

File: docs/api-docs/storefront/graphql/graphql-api-examples.mdx [hunk 2]
```diff
@@ -4,12 +4,47 @@ keywords: storefront, graphql
...
+For more information on formatting the request payload, see [the query example section of the GraphQL Storefront API Overview](/api-docs/storefront/graphql/graphql-storefront-api-overview#querying-within-a-bigcommerce-storefront).
+
+To use this API from a coupled storefront, use the following HTTP configuration:
+
+```js filename="Example query: Same-origin GraphQL Storefront HTTP configuration" showLineNumbers copy
+  fetch('/graphql', {
+    method: 'POST',
+    credentials: 'same-origin',
+    headers: {
+        'Accept': 'application/json',
```
Comment (bc-andreadao) [inline docs/api-docs/storefront/graphql/graphql-api-examples.mdx:26]: See comment below

**Review by bc-andreadao — COMMENTED** [review]
I added a comment, otherwise looks good

**Comment by slsriehl** [comment]
Accepted edits from @bc-andreadao, merging.

**Pull request merged**